src/sinks/vector/config.rs

```diff
@@ -224,6 +224,9 @@ fn new_client(
     });
 
     let mut proxy = ProxyConnector::new(https).unwrap();
+    // Make proxy connector aware of user TLS settings by setting the TLS connector:
```

Comment:
```suggestion
    // Make proxy connector aware of user TLS settings by setting the TLS connector.
```

I think the ":" was reference to the GH issue linked in `src/http.rs`

Reply:
ignore this... see my new comment